adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/ReleaseVersionResolver.kt

```diff
@@ -19,6 +21,25 @@ object ReleaseVersionResolver {
         }
     }
 
+    private const val DATA_FILE_URL = "https://raw.githubusercontent.com/M-Davies/openjdk-build/obsolete_releases/docs/data.json" // TODO Change me
```

Comment:
Not sure if we want a static file over the web for this, I am wondering if we could add something to the existing variants.json file, or maybe  just calculate what is obsolete

Reply:
I don't like it either but I didn't really get anywhere with finding a public API which does something similar.

Ooo I didn't realise variants.json existed, that would seem like a better place for it that openjdk-build. As for calculating, I'm not sure what you mean?

Reply:
@johnoliver Can you elaborate what you mean by calculating which JDK's are obsolete?

Reply:
> add something to the existing variants.json file

This has been done